src/lib/components/Access/utils.js

```diff
@@ -44,27 +39,27 @@ export function filesSection(filesStyle, filesContent) {
   );
 }
 
-
-export function MessageSection({intent, icon, title, text}) {
+export function MessageSection({ intent, icon, title, text }) {
   return (
     <Message visible {...intent}>
       <strong>
         <Icon name={icon} /> {title}
       </strong>
       {/* Needed to override semantic-ui's Card styling */}
-      <p style={{marginTop: "0.25em"}}>{text}</p>
+      <p style={{ marginTop: '0.25em' }}>{text}</p>
     </Message>
-  )
+  );
 }
 
-
-export function embargoSection(embargo) {
-  const fmtDate = (
-    embargo.date
-    ? DateTime.fromISO(embargo.date)
-      .toLocaleString(DateTime.DATE_FULL) // e.g. June 21, 2021
-    : "???"
-  );
+export function embargoSection(initialAccessValues, embargo) {
+  const fmtDate = initialAccessValues?.embargo?.until
```

Comment:
minor: it would be nice to check how babel will convert the `?` operator: if it is adding `if`s around, I would do it once at the beginning and then avoid it later on...

Reply:
Hmm, we trully only need to check for `initialAccessValues.embargo` since `initialAccessValues` should not be null, added it just in case, but will remove it since we only need to check for the `embargo`